### PR TITLE
Create Add DiscountRate.md

### PR DESCRIPTION
**Pull request by markborkum**
#### Any background context you want to provide?

N/A

#### What does this PR do?

New proposal to add "Discount Rate" element.

#### How should this be manually tested?

N/A

#### What are the relevant tickets?

N/A

### CODE REVIEW

File: BuildingSync.xsd [hunk 1]
```diff
@@ -4564,7 +4564,7 @@
         </xs:annotation>
         <xs:complexType>
           <xs:simpleContent>
-            <xs:extension base="xs:decimal">
+            <xs:extension base="xs:nonNegativeInteger">
```
Comment (nllong) [inline BuildingSync.xsd:4567]: perfect

File: BuildingSync.xsd [hunk 2]
```diff
@@ -4777,6 +4777,20 @@
...
+        </xs:annotation>
+        <xs:complexType>
+          <xs:simpleContent>
+            <xs:extension base="xs:decimal">
+              <xs:attribute ref="auc:Source"/>
+            </xs:extension>
+          </xs:simpleContent>
+        </xs:complexType>
+      </xs:element>
+      <xs:element name="DiscountRate" minOccurs="0">
```
Comment (nllong) [inline BuildingSync.xsd:4790]: Nice! And thanks for updating the documentation of the discount factor.

**Review by nllong — APPROVED** [review]

File: BuildingSync.xsd (hunk 2 above)
Comment (laurenliz22) [reply BuildingSync.xsd:4790]: Agreed!

File: proposals/2022/Add DiscountRate.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+## Justification
+
+### Present Value Calculation
+
+The equation for net present value is:
+
+$$ NPV\left({i,N}\right) = \sum_{t=0}^{N}{\frac{R_{t}}{\left({1+i}\right)^t}} $$
+
+where
+* $i$ is the discount rate (percentage).
```
Comment (laurenliz22) [inline proposals/2022/Add DiscountRate.md:16]: Should we say i is the interest rate here since  v is the discount rate where v=1/(1+i)?

**Review by laurenliz22 — APPROVED** [review]
Looks good, thanks Mark!  I had one comment on the definition of i

**Review by JieXiong9119 — APPROVED** [review]

**Pull request merged**